### PR TITLE
Standardization of layer gradient examples

### PR DESCRIPTION
**Pull request by Shashi456**
#499

### CODE REVIEW

**Comment by Shashi456** [comment]
Except for convolution and batchnorm, all other layers are done using tf.keras.

We approach the initialization of convolution layers differently, while the functionality remains at par.

**Comment by dan-zheng** [comment]
This is huge, thanks @Shashi456! 🙂
